02-basiccpp/README.md

```diff
@@ -0,0 +1,110 @@
+
+### Exercise 1 - Temperatures
+
+In `lib/01_temperatures.cpp` there are two functions, `celsiusToKelvin` and `celsiusToFahrenheit`. Both take a `double` and return another `double`.
+
+> From Celsius to Kelvin: `K = C + 273.15`
+ 
+> From Celsius to Fahrenheit: `F = C * 9 / 5 + 32`
+
+### Exercise 2 - Pancakes
+
+In `lib/02_pancakes.hpp` you can see a struct containing information about how many pancakes a particular person had for breakfast.
+
+```cpp
+struct PancakesEatenBy {
+  std::string name;
+  int num_pancakes_eaten;
+};
+```
+
+In particular, each object of type `PancakesEatenBy` has two attributes:
+
+1. A string containing the name of the person, and
+2. an integer representing the number of pancakes eaten by that person.
+
+In the source file (`lib/02_pancakes.cpp`) you can find a function `mostPancakesEaten`.
+
+> Modify it so that, given a vector of `PancakesEatenBy` objects, it returns the the one where the number of eaten pancakes is the highest.
+
+Below this, you can see another function, `moreThanNPancakesEaten`, which not only takes a vector of `PancakeEaten` objects, but also an integer `n`.
+
+> Modify it so that it returns a vector of those `PancakeEaten` objects where the number of pancakes eaten is strictly greater than `n`.
+
+### Exercise 3 - Vectors
+
+In `lib/03_vectors.cpp` there are four functions `sum`, `absoluteSum`, `countNegatives` and `average`. Fill out the implementation for all 4 functions
+
+> Modify the `sum` function implementation so that it returns the sum of all the elements of the vector.
+
+You may want to use range-based `for`-loops:
+
+```cpp
+const std::vector<double> v = {1, 2, 3, 4, 5};
+for (const double x : v) {
+    // in each iteration, x will take the value of the next element of v
+}
+```
+
+The next function in the file is called `absoluteSum`.
+By absolute sum we mean the sum of the absolute values of all the elements.
+
+> Modify `absoluteSum` so that it computes the absolute sum of all the elements of the given vector.
+
+You may find the [abs](https://www.cplusplus.com/reference/cmath/abs/) function from the C++ standard library helpful.
+Note that you will have to include the appropriate header file! (see the example on the linked page)
+
+The next function in the file is called `countNegatives`.
+It also takes a vector of floating point numbers, but it returns an unsigned integer.
+
+> Modify it so that it returns the number of negative elements in the vector.
+
+The last function in the file is called `average`. 
+It takes a vector of floating point numbers, and returns the average.
+
+> Modify it so that it returns the average of all elements in the vector.
+
+### Exercise 4 - Prime Number
+
+In `lib/04_isprime.cpp`, there's a function `isPrime` that takes an integer `n` and returns a boolean indicating whether the number is a prime or not.
+
+> Prime numbers are numbers that are only divisable by itself and 1.
```

Comment:
Just to be extra explicit - I'd say either point out that the numbers are > 1 or say numbers that have only two divisors (itself and one). Just to prevent anyone who wants to debate 1 being a prime.